src/robusta/core/model/base_params.py

```diff
@@ -41,8 +41,12 @@ class ResourceChartResourceType(Enum):
 class ActionParams(DocumentedModel):
     """
     Base class for all Action parameter classes.
+
+    :var platform_mute_finding: platform sink will process the alert but will not create a finding.
     """
 
+    platform_mute_finding: bool = False
```

Comment:
Is this flag relevant to all ActionParams?
Wouldn't it be better to add it in the EventParams?